WFInfo/Services/WarframeProcess/WarframeProcessFinder.cs

```diff
@@ -25,58 +25,36 @@ public Process Warframe
         private Process _warframe;
 
         private readonly IReadOnlyApplicationSettings _settings;
+        private System.Threading.Timer find_process_timer;
 
         public WarframeProcessFinder(IReadOnlyApplicationSettings settings)
         {
             _settings = settings;
+            find_process_timer = new System.Threading.Timer(FindProcess, null, 0, 40000);
         }
 
-        private void FindProcess()
+        private void FindProcess(Object stateInfo)
         {
-            // process already found
+            // Process was already found previously
             if (_warframe != null)
             {
-                if (_warframe.HasExited)
-                { //reset warframe process variables, and reset LogCapture so new game process gets noticed
-                    Main.dataBase.DisableLogCapture();
-                    _warframe.Dispose();
-                    _warframe = null;
-
-                    if (_settings.Auto)
-                        Main.dataBase.EnableLogCapture();
+                if (!_warframe.HasExited)
+                {
+                    return; // Current process is still good
                 }
-                else return; // Current process is still good
             }
 
+            //Main.AddLog("FindProcess have been triggered");
+
+            Process _identified_process = null;
```

Comment:
Granted we're very inconsistent with naming, but convention is to use `_` or `m_` prefix on private fields. This is not a private field, so it should be just `identified_process`